@app/server/src/plugins/SubscriptionsPlugin.ts

```diff
@@ -79,14 +60,36 @@ const SubscriptionsPlugin = makeExtendSchemaPlugin((build) => {
         """
         Triggered when the logged in user's record is updated in some way.
         """
-        currentUserUpdated: UserSubscriptionPayload @pgSubscription(topic: ${embed(
-          currentUserTopicFromContext
-        )})
+        currentUserUpdated: UserSubscriptionPayload
```

Comment:
The `@pgSubscription` directive is no more, instead write a `subscribePlan` (see below).